.gitlab-ci.yml

```diff
@@ -111,6 +129,32 @@ test:smoketests:linux:
     - ./mender-artifact-linux validate test-rfs.mender
     - make build
 
+test:coverage:linux:
+  stage: test
+  image: golang:1.18-buster
+  needs:
+    - job: build:coverage
+      artifacts: true
+  variables:
+    COVERAGE_FILE: coverage-linux-pkcs.txt
+  before_script:
+    - ./tests/test_sign_with_hsm/test_sign_with_hsm.sh --setup
+    - export TEST_MENDER_ARTIFACT_PATH=./mender-artifact-linux-coverage
+  script:
+    - ./tests/test_sign_with_hsm/test_sign_with_hsm.sh
+  after_script:
+    - set -- /tmp/coverage-artifact*.out
+    - head -n 1 $1 > ${COVERAGE_FILE}
+    - |
+      for cover in $@; do
```

Comment:
Q: Is it not just as easy to loop over the results from the wildcard match above here?

Reply:
It's also more about the aesthetics. Generally speaking we don't need `mode` in this file, but if it's cutted from a context and someone downloads an artifact, then it will be properly formatted.

Reply:
Also, what happends if there are more than `9` `coverage-artifact*.out` files?

Reply:
It just works 🙃